Cargo.toml

```diff
@@ -33,6 +33,7 @@ h2 = { version = "0.3.9", optional = true }
 itoa = "1"
 tracing = { version = "0.1", default-features = false, features = ["std"] }
 pin-project-lite = "0.2.4"
+pin-project = "1.0.12"
```

Comment:
Is it possible to get away with just using `pin-project-lite`? `pin-project` takes a bit longer to compile...

Reply:
I just used `pin-project` for easier development and wanted to switch to `pin-project-lite` once finished.

Reply:
Done